src/index.njk

```diff
@@ -137,9 +147,9 @@ eleventyNavigation:
     <p class="h3 section__header__hl">We’ve got your apps covered.</p>
 
     <p class="section__header__text">
-      Many FreeBSD ports and apps, Linux apps, and even trivial Darwin
-      &amp; macOS binaries can run on ravynOS. And we’re actively working on
-      improving it!
+      Soon we hope to have many apps supported on our system, 
+      but currently only native ravynOS applications are working with our new windowserver.
+      We hope to soon support the following:
```

Comment:
Remove references to Qt under "ravynOS, Always".

Change the text "FreeBSD ports that support Wayland continue to work on ravynOS right out of the box, or with minor changes." below to read "FreeBSD Ports that run on Wayland should run on ravynOS using an emulation layer."

Change the text "Trivial macOS and Darwin applications" below to read "Some macOS and Darwin applications".